topydo/ui/ColumnLayout.py

```diff
@@ -41,15 +41,18 @@ def _get_column_dict(p_cp, p_column):
 
     cp = configparser.RawConfigParser(defaults)
     files = [
-        "/etc/topydo_columns.conf",
-        home_config_path('.config/topydo/columns'),
-        home_config_path('.topydo_columns'),
         ".topydo_columns",
```

Comment:
Very minor, lines 44-46 should also be reversed, so the various extensions/variants of the names are evaluated in the same order as the main configuration file (Config.py lines 125 - 130).